pynicotine/pluginsystem.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, core, config):
         self.core = core
         self.config = config
 
-        log.add_debug("Loading plugin handler")
+        log.add("Loading plugin system")
```

Comment:
Ideally, this log line would only appear if Preferences > 'Enable logging' is True

Reply:
You can probably move it after line 262.

Reply:
Thanks, good idea. Actually I moved it just above as this seems to make sense. See comment below.